lib/iris/fileformats/grib/__init__.py

```diff
@@ -767,6 +767,7 @@ def load_pairs_from_fields(grib_messages):
 
         >>> from iris.fileformats.grib import load_pairs_from_fields
```

Comment:
Could you also remove this import? I think it just confuses things.

This example uses `filename` and `GribMessage` which are defined/imported in the above example as well as  `load_pairs_from_fields` which is imported in both places.

Either reimport/define everything so that this second example looks like:
```
        >>> from iris.fileformats.grib import load_pairs_from_fields
        >>> from iris.fileformats.grib.message import GribMessage
        >>> filename = iris.sample_data_path('polar_stereo.grib2')
        >>> cleaned_messages = GribMessage.messages_from_filename(filename)
        >>> cleaned_messages = list(cleaned_messages)
        >>> for message in cleaned_messages:
        ...     if message.sections[1]['productionStatusOfProcessedData'] == 0:
        ...         message.sections[1]['productionStatusOfProcessedData'] = 4
        >>> cubes = load_pairs_from_fields(cleaned_messages)
```

Or treat it like a continuation of the previous example
```
        >>> cleaned_messages = GribMessage.messages_from_filename(filename)
        >>> cleaned_messages = list(cleaned_messages)
        >>> for message in cleaned_messages:
        ...     if message.sections[1]['productionStatusOfProcessedData'] == 0:
        ...         message.sections[1]['productionStatusOfProcessedData'] = 4
        >>> cubes = load_pairs_from_fields(cleaned_messages)
```